tests/integration/store/test_version_store.py

```diff
@@ -1607,3 +1624,74 @@ def test_can_write_tz_aware_data_series(library):
     read_data = read_data.dt.tz_localize('UTC').dt.tz_convert(read_data.index.tzinfo)
     assert library._versions.find_one({'symbol': 'symTzSer'})['type'] == PandasSeriesStore.TYPE
     assert_series_equal(myseries, read_data)
+
+
+@pytest.mark.parametrize('write_cfg, read_cfg, append_cfg, reread_cfg', [
+    (FwPointersCfg.DISABLED, FwPointersCfg.DISABLED, FwPointersCfg.DISABLED, FwPointersCfg.DISABLED),
+    (FwPointersCfg.ENABLED, FwPointersCfg.ENABLED, FwPointersCfg.ENABLED, FwPointersCfg.ENABLED),
+    (FwPointersCfg.HYBRID, FwPointersCfg.HYBRID, FwPointersCfg.HYBRID, FwPointersCfg.HYBRID),
+
+    (FwPointersCfg.HYBRID, FwPointersCfg.DISABLED, FwPointersCfg.HYBRID, FwPointersCfg.DISABLED),
+    (FwPointersCfg.HYBRID, FwPointersCfg.ENABLED, FwPointersCfg.HYBRID, FwPointersCfg.ENABLED),
+
+    (FwPointersCfg.ENABLED, FwPointersCfg.HYBRID, FwPointersCfg.ENABLED, FwPointersCfg.HYBRID),
+    (FwPointersCfg.DISABLED, FwPointersCfg.HYBRID, FwPointersCfg.DISABLED, FwPointersCfg.HYBRID),
+
+    (FwPointersCfg.DISABLED, FwPointersCfg.ENABLED, FwPointersCfg.DISABLED, FwPointersCfg.ENABLED),
+    # This throws exception, and ENABLED --> DISABLED fw-pointers config is not allowed/supported
```

Comment:
This will almost certainly happen in real life though if disabled is the default

Reply:
my last commit solved all compatibility issues for all possible transitions.
I will add even more thorough tests tomorrow, but the tests which were already failing now are green.